lib/std/http/Client.zig

```diff
@@ -538,8 +538,18 @@ pub const Request = struct {
 
     pub const StartError = Connection.WriteError || error{ InvalidContentLength, UnsupportedTransferEncoding };
 
+    pub const StartOptions = struct {
+        /// Specifies that the uri should be used as is
+        raw_uri: bool = false,
+    };
+
     /// Send the request to the server.
     pub fn start(req: *Request) StartError!void {
+        try req.startWithOptions(.{});
+    }
+
+    /// Send the request to the server.
+    pub fn startWithOptions(req: *Request, options: StartOptions) StartError!void {
```

Comment:
No need for this to be two variants, the current use case can be covered by `start(.{})` because we can provide sane defaults.

Reply:
I'll remove the "WithOptions" then. I notice that since 0.11.0, a fetch function was added. Should I leave that alone in favor of the defaults, or add a flag into FetchOptions as well?

Reply:
Might as well, it was intended to be a "fast" path through std.http and there's no reason not to.